From be58e066edb6452bc2b58cbc7bc84ebe70f33c6a Mon Sep 17 00:00:00 2001
From: Nikola Zivkovic <zivke85@gmail.com>
Date: Thu, 12 May 2022 11:21:38 +0200
Subject: [PATCH 1/3] Properly clear the master process data

The process data size never gets set to zero during master configuration clearing.
---
 lib/master.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/lib/master.c b/lib/master.c
index bab00345..3fb9c509 100644
--- a/lib/master.c
+++ b/lib/master.c
@@ -58,11 +58,6 @@ void ec_master_clear_config(ec_master_t *master)
     ec_domain_t *d, *next_d;
     ec_slave_config_t *c, *next_c;
 
-    if (master->process_data)  {
-        munmap(master->process_data, master->process_data_size);
-        master->process_data = NULL;
-    }
-
     d = master->first_domain;
     while (d) {
         next_d = d->next;
-- 
2.25.1

